cumulus/parachains/runtimes/assets/asset-hub-westend/src/xcm_config.rs

```diff
@@ -57,24 +57,30 @@ use xcm_builder::{
 use xcm_executor::{traits::WithOriginFilter, XcmExecutor};
 
 parameter_types! {
-	pub const WestendLocation: MultiLocation = MultiLocation::parent();
+	pub const WestendLocation: Location = Location::parent();
+	pub const WestendLocationV3: xcm::v3::Location = xcm::v3::Location::parent();
```

Comment:
This and other `XYZv3` constants are not used anywhere AFAICT

Reply:
Might have missed some that were not needed, but in general I kept these only for storage items since a storage migration would've been much worse to also bundle into this PR